data/helpers.d/php

```diff
@@ -45,7 +64,27 @@ ynh_add_fpm_config () {
 		chown root: "$finalphpini"
 		ynh_store_file_checksum "$finalphpini"
 	fi
-	ynh_systemd_action --service_name=$fpm_service --action=reload
+
+	# Create a config for a dedicated php-fpm service for the app
+	echo "
+[Unit]
+Description=PHP $phpversion FastCGI Process Manager for $app
+After=network.target
+
+[Service] 
+Type=notify
+PIDFile=/run/php/php${phpversion}-fpm-$app.pid
+ExecStart=/usr/sbin/php-fpm${phpversion} --nodaemonize --fpm-config $finalphpconf --pid /run/php/php${phpversion}-fpm-$app.pid
+ExecReload=/bin/kill -USR2 \$MAINPID
+
+[Install]
+WantedBy=multi-user.target
+" > ../conf/$fpm_service
+
+	# Create this dedicated php-fpm service
+	ynh_add_systemd_config --service=$fpm_service --template=$fpm_service
+
```

Comment:
```suggestion
	yunohost service add $fpm_service --log_type "systemd"
```

Reply:
Yes indeed.
But...
There's actually an error log in /var/log/php7.0-fpm.log and another part of the log in syslog.
Would be interesting to have everything in the same place.

Reply:
`php_admin_value[error_log]` and `php_admin_flag[log_errors]` are supposed to be the keys to use in the conf file.
But maybe only for the errors.

Meanwhile, I got the stop and start log into syslog and some info into /var/log/php-fpm.log...

Interesting fact, php-fpm.log is the default value for [php-fpm.conf](https://github.com/php/php-src/blob/master/sapi/fpm/php-fpm.conf.in#L24)
Looks like a recurrent problem, .d conf does not really overwrite the value, despite the doc saying it does...

To use a `/etc/php/7.0/dedicated-fpm/php-fpm.conf` does not change anything...

I do not see any proper solution...

Reply:
Finally succeed to have all logs into a specific file for each service.
I tried with success with 3 apps, each running on their dedicated service.

Here again, to manage the backup/restore, it would be a lot easier to run the helper again.